src/layman/map/rest_workspace_test.py

```diff
@@ -38,7 +38,7 @@
 
 METADATA_PROPERTIES_EQUAL = METADATA_PROPERTIES
 
-num_maps_before_test = 0  # pylint: disable=invalid-name
+num_of_publications = PublicationCounter()
```

Comment:
`num_of_publications` -> `PUBLICATION_COUNTER` (not sure, why pylint accepts it in lowercase)